docs/recipes/others/speed_dial.md

```diff
@@ -0,0 +1,471 @@
+```python exec
+import reflex as rx
+```
+
+# Speed Dial
+
+A speed dial is a component that allows users to quickly access frequently used actions or pages. It is often used in the bottom right corner of the screen.
+
+# Vertical
+
+```python demo exec toggle
+class SpeedDialVertical(rx.ComponentState):
+	is_open: bool = False
+
+	def toggle(self, value: bool):
+		self.is_open = value
+
+	@classmethod
+	def get_component(cls, **props):
+		def menu_item(icon: str, text: str) -> rx.Component:
+			return rx.tooltip(
+				rx.icon_button(
+					rx.icon(icon, padding="2px"),
+					variant="soft",
+					color_scheme="gray",
+					size="3",
+					cursor="pointer",
+					radius="full",
+				),
+				side="left",
+				content=text,
+			)
+
+		def menu() -> rx.Component:
+			return rx.vstack(
+				menu_item("copy", "Copy"),
+				menu_item("download", "Download"),
+				menu_item("share-2", "Share"),
+				position="absolute",
+				bottom="100%",
+				spacing="2",
+				padding_bottom="10px",
+				left="0",
+				direction="column-reverse",
+				align_items="center",
+			)
+
+		return rx.box(
+			rx.box(
+				rx.icon_button(
+					rx.icon(
+						"plus",
+						style={
+							"transition": "transform 150ms cubic-bezier(0.4, 0, 0.2, 1)",
+						},
+						class_name="dial",
+					),
+					variant="solid",
+					color_scheme="blue",
+					size="3",
+					cursor="pointer",
+					radius="full",
+					position="relative",
+				),
+				rx.cond(
+					cls.is_open,
+					menu(),
+				),
+				position="relative",
+				style={
+					"&:hover .dial": {
+						"transform": "rotate(45deg)",
+						"transition": "transform 150ms cubic-bezier(0.4, 0, 0.2, 1)",
+					},
+				},
```

Comment:
if you take the previous suggestion, then the rotation state of the element should be based on the value of `is_open`, rather than whether the element is hovered.

So, remove this style dict and add the conditional transform directly to the `icon` component:

```python
rx.icon(
    "plus",
    style={
        "transform": rx.cond(cls.is_open, "rotate(45deg)", "rotate(0)"),
        "transition": "transform 150ms cubic-bezier(0.4, 0, 0.2, 1)",
    },
),
```